src/Common/DTOs/AddressTypes/Address.cs

```diff
@@ -0,0 +1,19 @@
+namespace SSW.Rewards.Shared.DTOs.AddressTypes;
+
+public class Address
+{
+    public string streetNumber { get; set; }
```

Comment:
We have so many of these warnings, is there a way that we can ignore or fix these?

Reply:
As per our conversation, we will address this separately. Task captured in #763.